term_windows.go

```diff
@@ -71,19 +71,19 @@ func StdStreams() (stdIn io.ReadCloser, stdOut, stdErr io.Writer) {
 	// go-ansiterm hasn't switch to x/sys/windows.
 	// TODO: switch back to x/sys/windows once go-ansiterm has switched
 	if emulateStdin {
-		stdIn = windowsconsole.NewAnsiReader(windows.STD_INPUT_HANDLE)
+		stdIn = windowsconsole.NewAnsiReader(int64(windows.STD_INPUT_HANDLE))
```

Comment:
I think the linter currently only runs on Linux, so probably is not complaining because of that.

Wondering if we need to add a ` // nolint: unconvert` to these (also to make it more clear that this is intentional)

```suggestion
		stdIn = windowsconsole.NewAnsiReader(int64(windows.STD_INPUT_HANDLE)) // nolint: unconvert
```

(open to suggestions 😅)

Reply:
Sure, added the `nolint` to make it clear about the intention.